scss/_patterns_rule.scss

```
@@ -0,0 +1,21 @@
+@import 'settings';
+
+$small-offset: map-get($grid-margin-widths, small);
+$default-offset: map-get($grid-margin-widths, default);
+
+@mixin vf-p-rule {
+  .p-rule {
+    @extend %hr;
+    @extend %u-no-margin--bottom--hr;
+    max-width: calc($grid-max-width - 2 * $small-offset);
+
+    @media (min-width: $threshold-4-6-col) {
+      max-width: calc($grid-max-width - 2 * $default-offset);
+    }
+
+    &--highlight {
+      @extend %u-no-margin--bottom--hr;
+      @include vf-highlight-bar($color-dark, bottom, false);
```

@ClementChaumel I guess all we need is change the position here to top (which is default) - but I'm not 100% sure, so please verify.

```suggestion
      @include vf-highlight-bar($color-dark);
```